daniel_sonner.cfdg

```diff
@@ -0,0 +1,12 @@
+startshape BOARD 
+
+rule BOARD {
```

Comment:
Very nice chess board!  I had not realized you could use multiply a rule by a position to get this effect! Very cool!